open-api/rest-catalog-open-api.yaml

```diff
@@ -1610,13 +1610,27 @@ components:
 
     PageToken:
       description:
-        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables). 
-        Clients will initiate the first paginated request by sending an empty `pageToken` e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
-        signaling to the service that the response should be paginated.
+        An opaque token that allows clients to make use of pagination for list APIs
+        (e.g. ListTables). Clients may initiate the first paginated request by sending an empty
+        query parameter `pageToken` to the server e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
```

Comment:
I don't think this should have multiple "examples" that conflict with one another. If an example is necessary (and I don't think that it is) then there should be only one recommendation in a spec.

Reply:
Sounds good, will include one example then with the `=` form, based on your recommendation from prior pagination pr.

Reply:
To be clear, I prefer removing these examples because this is an area that should be covered by the OpenAPI spec. The purpose of OpenAPI is to allow us to specify these things without ambiguity. I would prefer relying on that rather than giving an example here at all.

But if you want to have an example, there should be only one example.

Reply:
I think your point is fair I only see one example of this in the rest spec https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml#L198

So for now will remove these examples based on your feedback.

Reply:
@rahil-c I did see those examples too and I think the difference there is there may be some confusion about how namespaces are encoded where as here, an empty query parameter really isn't ambiguous and is covered by standard URL specification.